### PR TITLE
A blog post about the community team work

### PR DESCRIPTION
**Pull request by skade**
Corrections, additions, etc. appreciated!

Still missing links.

[Rendered](https://github.com/rust-community/blog.community.rs/blob/5c8331c6c9c0263c11f81681bce561702e59f377/_posts/2017-04-03-about-the-rust-community-team.md)

### CODE REVIEW

File: _posts/2017-04-03-about-the-rust-community-team.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The [roster](https://www.rust-lang.org/en-US/team.html#Community-team) has changed expanded since, especially with organisers of meetups and conferences joining.
+
+Most members of the team did community work in other communities, which certainly gave us a headstart in some aspects. Still, the only requirement to participate is interest in our work and participation.
+
+At the same time, the still-young Rust project was in a sense a blank slate, making it very easy to evolve the structure of community work: it was only just forming. For me, personally, that was a very refreshing experience.
+
+The community team is not the [❤️moderation team❤️](), although there is some personal overlap: helping out with discussions is not our
```
Comment (Manishearth) [inline _posts/2017-04-03-about-the-rust-community-team.md]: ....

File: _posts/2017-04-03-about-the-rust-community-team.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The [roster](https://www.rust-lang.org/en-US/team.html#Community-team) has changed expanded since, especially with organisers of meetups and conferences joining.
+
+Most members of the team did community work in other communities, which certainly gave us a headstart in some aspects. Still, the only requirement to participate is interest in our work and participation.
+
+At the same time, the still-young Rust project was in a sense a blank slate, making it very easy to evolve the structure of community work: it was only just forming. For me, personally, that was a very refreshing experience.
+
+The community team is not the [❤️moderation team❤️](), although there is some personal overlap: helping out with discussions is not our 
+
+## Setup: We do FOSS community
```
Comment (Manishearth) [inline _posts/2017-04-03-about-the-rust-community-team.md]: community management?

File: _posts/2017-04-03-about-the-rust-community-team.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+We are aware that backchannels in open communities are always a matter of trust, and the backchannel has the same rules as all other Rust venues, especially the [Code of Conduct](https://www.rust-lang.org/en-US/conduct.html).
+
+## In Rust, Community is Core
+
+In the Rust project, community work is seen as parallel and as important as other contributions. For some of our members, it is the main contribution to the project, not a side issue to their coding work. This has drawn in a lot of people that have a lot of experience.
+
+Of all the Rust project teams, the community team is probably the one where the team work happens mostly outside of the projects organisation. We talk to interested parties, library maintainers, spend our time at meetups (often our own), write documentation. This isn't just about hanging around at tables - you need to be very well-informed about all aspects of the Rust projects and its workings, as you might have to point people towards their areas of interest.
```
Comment (Manishearth) [inline _posts/2017-04-03-about-the-rust-community-team.md]: (the docs team writes documentation, we write a different kind, might be worth qualifying that word there)

File: _posts/2017-04-03-about-the-rust-community-team.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+The Rust Community team is a volunteer effort. Even the Mozilla employees on the team are volunteering next to their job. Obviously, there are sometimes connections to their day jobs.
+
+This, most importantly, means that the most constrained resource of the team is time. We have find relatively lightweight and easy to pick up processes. Still, documentation and coordination is important, which introduces work. We believe though, that - in total - we reduce work for individuals. It is the express goal of the team to be supportive to projects, not introduce more effort to them.
+
+Still, FOSS work is subject to the tides of life much more then a proper job. We all have our lives with ups and downs. While we try our best to be reliable, always keep this in mind should an interaction with us does not end up perfect or delayed. But still, we want to be measured by your expectations and not disappoint you.
+
+If you are a member of another FOSS community, treat our work with the same attitude you have to other projects code: with respect, but with the willingness to take, be inspired, ask for advice and adopt!
+
```
Comment (Manishearth) [inline _posts/2017-04-03-about-the-rust-community-team.md]: Some mention of switchboarding and also the production user calls?

Both are functions we do that not everyone is aware of (but folks can benefit from being aware of them)

**Review by Manishearth — APPROVED** [review]

File: _posts/2017-04-03-about-the-rust-community-team.md (hunk 1 above)
Comment (skade) [reply _posts/2017-04-03-about-the-rust-community-team.md]: Oh, will be fixed soon

File: _posts/2017-04-03-about-the-rust-community-team.md (hunk 4 above)
Comment (skade) [reply _posts/2017-04-03-about-the-rust-community-team.md]: There's another blog post of all tiny things coming up.( basically a port of my lobste.rs comment)

File: _posts/2017-04-03-about-the-rust-community-team.md (hunk 3 above)
Comment (skade) [reply _posts/2017-04-03-about-the-rust-community-team.md]: good catch!

File: _posts/2017-04-03-about-the-rust-community-team.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+In this post, I'll explain how we are set up and how you can follow our work.
+
+## A very short history
+
+The Rust Community IRC channel was created by me after feeling the need to get in touch with other Meetup organisers. I ran the [Berlin Rust Hack and Learn](https://www.meetup.com/de-DE/opentechschool-berlin/) and slowly, there were other events popping up.
+
+Shortly after launch of Rust 1.0, the Rust Community team was founded and [announced](https://internals.rust-lang.org/t/announcing-the-community-subteam/2248) by [Steve Klabnik](https://github.com/steveklabnik) as a way to bring representation of community workers into the Rust project. See then announcement for the initial goals. Later, [Erick Tryzelaar](https://github.com/erickt) took over the lead. After 2 years of active work, we have at some points diverged from this and extended it, but many still hold true.
+
+The [roster](https://www.rust-lang.org/en-US/team.html#Community-team) has changed expanded since, especially with organisers of meetups and conferences joining.
```
Comment (sebasmagri) [inline _posts/2017-04-03-about-the-rust-community-team.md]: It would be just `expanded` IMO

Comment (skade) [reply _posts/2017-04-03-about-the-rust-community-team.md]: Thanks!

File: _posts/2017-04-03-about-the-rust-community-team.md (hunk 4 above)
Comment (badboy) [reply _posts/2017-04-03-about-the-rust-community-team.md]: > more **than** a proper job

**Comment by badboy** [comment]
:shipit:

File: _posts/2017-04-03-about-the-rust-community-team.md (hunk 4 above)
Comment (skade) [reply _posts/2017-04-03-about-the-rust-community-team.md]: corrected

**Pull request merged**